dynawo/CMakeLists.txt

```diff
@@ -272,6 +272,7 @@ if(BUILD_TESTS OR BUILD_TESTS_COVERAGE)
   add_library(dynawo_Test INTERFACE)
   target_link_libraries(dynawo_Test INTERFACE
     dynawo_Common
+    dynawo_Test_utils
```

Comment:
isn't necessary only in case of BUILD_TESTS or BUILD_TESTS_COVERAGE ?

Reply:
you are right, it is protected above (line 253)